### DISCUSSION TITLE
Mappable Key bindings

### DISCUSSION THREAD

**Discussion by SayedDileri** [minbrowser/min#2028] [Ideas]
I really love this browser. Simple, clean and free of any distractions. However, I have recently been using VIM for chrome extension. I would really like a extension or setting that would allow me to accomplish that in Min Browser. 

![image](https://user-images.githubusercontent.com/89445591/175760178-76749782-50e8-4d7e-8185-5698bd743ba7.png)

**Comment by PalmerAL** [comment]
This exists! https://github.com/PalmerAL/min-vim-mode
